### ISSUE TITLE
Error in version 1.1.1

### ISSUE THREAD

**Issue by kaermorchen** [DockYard/ember-changeset#106]
Hello.

This my code:

```
      const changeset = this.get('changeset');
      const model = this.get('model');
      const validation = this.get('validation');

      return changeset
        .cast(keys(validation))
        .validate()
        .then(() => {
          if (changeset.get('isValid')) {
            return changeset.execute();
          }
        });

```

I catched an error - `You must use Ember.set() to set the `_changes` property`.

![2016-09-05_15-17-03](https://cloud.githubusercontent.com/assets/11972062/18247558/ef5a6fb0-737b-11e6-82f9-b90ad4ccab11.png)

This is cast function

![2016-09-05_15-25-18](https://cloud.githubusercontent.com/assets/11972062/18247725/d266a97c-737c-11e6-8cbc-fbaf2e2c57a4.png)

I use:
```
DEBUG: -------------------------------
DEBUG: Ember             : 2.7.1
DEBUG: Ember Data        : 2.7.0
DEBUG: jQuery            : 2.2.4
DEBUG: Ember Simple Auth : 1.1.0
DEBUG: -------------------------------
```

**Comment by poteto** [comment]
Thanks, let me look into it

**Comment by poteto** [comment]
Fixed in `1.1.2`
